### PR TITLE
Global search changes

### PR DESCRIPTION
**Pull request by Uditgulati**
#### Description

Optimised the phenotype global search.

### CODE REVIEW

File: wqflask/wqflask/gsearch.py [hunk 1]
```diff
@@ -228,16 +230,22 @@ def __init__(self, kw):
...
-                    if not trait_ob:
-                        continue
                     if this_trait['dataset'] == this_trait['group'] + "Publish":
-                      try:
-                        if trait_ob.locus_chr != "" and trait_ob.locus_mb != "":
-                            this_trait['max_lrs_text'] = "Chr" + str(trait_ob.locus_chr) + ": " + str(trait_ob.locus_mb)
-                      except:
-                          this_trait['max_lrs_text'] = "N/A"
+                        try:
+                            if this_trait['locus_chr'] != None and this_trait['locus_mb'] != None:
```
Comment (BonfaceKilz) [inline wqflask/wqflask/gsearch.py]: **nitpick** These are equivalent:

```
if this_trait['locus_chr'] != None and this_trait['locus_md'] != None:
...
```

```
if this_trait.get("locus_chr") and this_trait.get("locus_md"):
...
```

File: wqflask/wqflask/gsearch.py [hunk 2]
```diff
@@ -228,16 +230,22 @@ def __init__(self, kw):
...
+                                this_trait['max_lrs_text'] = "Chr" + str(this_trait['locus_chr']) + ": " + str(this_trait['locus_mb'])
```
Comment (BonfaceKilz) [inline wqflask/wqflask/gsearch.py]: **nitpick:** It'd be nice to use f-strings which make things readable IMHO.

**Review by BonfaceKilz — APPROVED** [review]
Nothing out of the ordinary. Although I didn't quite follow along with the SQL query change. From a high-level overview, I like that you got rid of the regexes.

**Review by zsloan — APPROVED** [review]
This works well for me when I tested, so it should be fine to merge.

**Comment by pjotrp** [comment]
@Uditgulati what is the status of this PR?

**Review by BonfaceKilz — APPROVED** [review]

**Pull request merged**